save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/utils/LocalDateTimeUtils.kt

```diff
@@ -0,0 +1,29 @@
+@file:Suppress("MagicNumber")
+
+/**
+ * Utilities for LocalDateTime
+ */
+
+package com.saveourtool.save.utils
+
+/**
+ * @param data: year, month, day
+ * @return [data] in LocalDateTime format
+ */
+fun createLocalDateTime(data: Triple<Int, Int, Int>): LocalDateTime {
+    val year = data.first
+    val month = data.second + 1
+    val day = data.third
+    return LocalDateTime.of(year, month, day, 0, 0, 0)
```

Comment:
You can use `LocalDate` instead of `LocalDateTime` if you care only about date

Reply:
We use LocalDateTime in our execution. In addition, I decided to leave it for the future.